server-data/resources/[framework]/rsg-appearance/client/cl_main.lua

```diff
@@ -242,11 +242,23 @@ AddEventHandler('rsg-appearance:LoadSkinClient', function()
     end
 end)
 
-RegisterCommand('loadskin', function(source, args, raw)
-    if isLoggedIn then
-        TriggerServerEvent("rsg-appearance:LoadSkin")
-    end
-end)
+local function reloadSkin()
+    if not isLoggedIn then return end
+    TriggerServerEvent("rsg-appearance:LoadSkin")
+end
+
+local RELOAD_SKIN_COMMANDS = {
+    'loadskin',
+    'reloadskin',
+    'refreshskin',
+    'rc'
+}
+
+for _, command in ipairs(RELOAD_SKIN_COMMANDS) do
```

Comment:
```suggestion
for _key, command in ipairs(RELOAD_SKIN_COMMANDS) do
```
For code readability, consider using variable name with prefix instead of underscore.